.github/workflows/deploy-to-eks.yaml

```
@@ -64,7 +67,10 @@ jobs:
           K8S_VERSION=${{ inputs.k8s_version || env.K8S_VERSION }}
           echo "K8S_VERSION=${K8S_VERSION}" >> $GITHUB_ENV
           UATS_BRANCH=${{ inputs.uats_branch || env.UATS_BRANCH }}
-          echo "UATS_BRANCH=${UATS_BRANCH}" >> $GITHUB_ENV
+          echo "UATS_BRANCH=${UATS_BRANCH}" >> $GITHUB_ENV          
+          BUNDLE_URL=${{ inputs.bundle_url || env.UATS_BRANCH }}
```

```suggestion
          BUNDLE_URL=${{ inputs.bundle_url || env.URL }}
```
?
Although it would make sense to have the same name everywhere (BUNDLE_URL vs URL)

---

Also, if we remove `BUNDLE_URL` from `dependencies.yaml`, I think we don't need this line. The next one suffices.